api/migrations/20200626194403-removeOrphanedDocs.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+var ObjectID = require('mongodb').ObjectID;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.connection.connect(db.connectionString, { native_parser: true })
+    .then(async (mClient) => {
```

Comment:
Convert to await

Reply:
3fb0c0014b44f40a4f53dd1c437c15124582029f